components/admin/lessons/AdminLessonInputs/AdminLessonInputs.tsx

```diff
@@ -18,36 +18,50 @@ import {
 } from '@apollo/client'
 
 type Module = { id: number; name: string; content: string; order: number }
+type Challenge = {
```

Comment:
There should already be a `Challenge` type set somewhere that you can use

Reply:
I'll look into it.